xcode/Subconscious/Shared/Services/DataService.swift

```diff
@@ -28,6 +28,12 @@ enum DataServiceError: Error, LocalizedError {
     }
 }
 
+/// Record of a successful move transaction
```

Comment:
We were previously not including enough information on successful move operations. MoveReceipt is a struct that contains all the crucial details of a move. We return it on successful  move.